docs/api/users.rst

```diff
@@ -92,105 +87,19 @@ Response
 .. code-block:: json
 
     {
-        "credentials_expired": false,
-        "email": "chelsie.witting@example.com",
-        "email_canonical": "chelsie.witting@example.com",
-        "enabled": true,
-        "expired": false,
-        "groups": [],
-        "id": 481,
-        "locked": false,
-        "password": "EbOLtGHYxJKotA+bdb9BElhXPd8qZsnlo8CjDdCk+qFR22EEZJoOTntBX/M5GUXw2vnEqOKIEVPaJr66yxXqqQ==",
-        "roles": [],
-        "salt": "h9ltmmawvdsk08oocogkws4sg040k04",
-        "username": "chelsie.witting@example.com",
-        "username_canonical": "chelsie.witting@example.com"
+      "enabled": true,
+      "id": "@integer@",
+      "locked": false,
+      "roles": "@array@",
+      "username": "@string@",
```

Comment:
I am not sure I understand your question.
What I did:
- Removed fields that are not returned by the API anymore (the documentation was outdated)
- Changed the format `"username": "chelsie.witting@example.com"` into `"username": "@string@"` if everybody likes this format, it can be a step toward having automatic generation of the API documentation from the test cases.